packages/grafana-ui/src/components/Table/TableNG/Cells/PillCell.test.tsx

```diff
@@ -20,69 +19,42 @@ describe('PillCell', () => {
     restoreThemeContext();
   });
 
-  const mockCellOptions: TablePillCellOptions = {
-    type: TableCellDisplayMode.Pill,
-  };
-
-  const mockField: Field = {
+  const fieldWithValues = (values: unknown[]): Field => ({
     name: 'test',
     type: FieldType.string,
-    values: [],
+    values: values,
     config: {},
-  };
-
-  const mockFrame: DataFrame = {
-    name: 'test',
-    fields: [mockField],
-    length: 1,
-  };
-
-  const defaultProps = {
-    field: mockField,
-    justifyContent: 'flex-start' as const,
-    cellOptions: mockCellOptions,
-    rowIdx: 0,
-    frame: mockFrame,
-    height: 30,
-    width: 100,
-    theme: {} as GrafanaTheme2,
-    cellInspect: false,
-    showFilters: false,
-  };
+  });
 
   const ser = new XMLSerializer();
 
   const expectHTML = (result: RenderResult, expected: string) => {
     let actual = ser.serializeToString(result.asFragment()).replace(/xmlns=".*?" /g, '');
-    expect(actual).toEqual(expected.replace(/^\s*|\n/gm, ''));
+    expect(actual).toContain(expected.replace(/^\s*|\n/gm, ''));
```

Comment:
I changed how this works so that these tests are agnostic of whether there's a wrapper div. maybe we can kill the wrapper div in the future, but for now we need it.